interop-cats/jvm/src/main/scala/zio/interop/catsjvm.scala

```diff
@@ -84,7 +84,7 @@ sealed trait CatsInstances1 extends CatsInstances2 {
     : MonadError[ZIO[R, E, ?], E] with Bifunctor[ZIO[R, ?, ?]] with Alternative[ZIO[R, E, ?]] =
     new CatsAlternative[R, E] with CatsBifunctor[R]
 
-  implicit def taskConcurrentInstances[R]: effect.Concurrent[TaskR[R, ?]] with SemigroupK[TaskR[R, ?]] =
+  implicit def ZIOtaskConcurrentInstances[R]: effect.Concurrent[TaskR[R, ?]] with SemigroupK[TaskR[R, ?]] =
```

Comment:
variable names do not impact implicit resolution. I doubt that change does anything about your original issue...

Reply:
It would be nice to have this change in a separate PR for clarity. Thought scope of this PR was to add to the resources page in the docs